src/databricks/labs/remorph/reconcile/connectors/snowflake.py

```diff
@@ -1,16 +1,67 @@
+from pyspark.errors import PySparkException
 from pyspark.sql import DataFrame
 
 from databricks.labs.remorph.reconcile.connectors.data_source import DataSource
+from databricks.labs.remorph.reconcile.constants import SourceDriver
 from databricks.labs.remorph.reconcile.recon_config import Schema, Tables
 
 
 class SnowflakeDataSource(DataSource):
```

Comment:
can you add test cases for these

Reply:
Yes. I have added test case for SnowflakeDataSource using moving. Since therent any services to strat a snowflake server on the fly.